### PR TITLE
general purpose info field in contract

### PR DESCRIPTION
**Pull request by yaronvel**

### CODE REVIEW

File: contracts/KyberNetwork.sol [hunk 1]
```diff
@@ -23,7 +23,7 @@ contract KyberNetwork is Withdrawable, Utils {
     FeeBurnerInterface    public feeBurnerContract;
     uint                  public maxGasPrice = 50 * 1000 * 1000 * 1000; // 50 gwei
     bool                  public enabled = false; // network is enabled
-    uint                  public networkState; // this is only a field for UI.
+    mapping(bytes32=>uint) public info; // this is only a UI field for external app.
```
Comment (ilanDoron) [inline contracts/KyberNetwork.sol]: don't understand why bytes32? 
what key type will be used?
how will we know per query the key?

File: web3deployment/deployer.js [hunk 2]
```diff
@@ -272,7 +273,22 @@ async function main() {
...
+    var srcString1 = web3.utils.sha3("src token " + (2*i).toString());
+    var destString1 = web3.utils.sha3("dest token " + (2*i).toString());
+    var srcString2 = web3.utils.sha3("src token " + (2*i + 1).toString());
+    var destString2 = web3.utils.sha3("dest token " + (2*i + 1).toString());
+
+    await sendTx(networkContract.methods.setInfo(srcString1, ethAddress));
+    await sendTx(networkContract.methods.setInfo(destString1, tokens[i]));
+    await sendTx(networkContract.methods.setInfo(srcString2, tokens[i]));
+    await sendTx(networkContract.methods.setInfo(destString2, ethAddress));
   }
```
Comment (ilanDoron) [inline web3deployment/deployer.js]: I am not sure why you are adding this.
My script does query which token listed per reserve in the network contract.

Comment (ilanDoron) [inline web3deployment/deployer.js]: I assume set info required also for network state - for the UI

File: contracts/KyberNetwork.sol (hunk 1 above)
Comment (yaronvel) [reply contracts/KyberNetwork.sol]: Network state will be listed as `info(sha3("network state")), or any other way. It is in application level, not contract level.

File: web3deployment/deployer.js (hunk 2 above)
Comment (yaronvel) [reply web3deployment/deployer.js]: Im token asked for it.

Comment (yaronvel) [reply web3deployment/deployer.js]: One we define it, yes.

**Comment by yaronvel** [comment]
This is something to settle with andrew and victor. For now, we ignore it.

**Pull request merged**